src/test/libraries/ArrayUtil.t.sol

```diff
@@ -195,6 +195,31 @@ contract ArrayUtilTest is Test {
         assertEq(indexes[4], 0);
     }
 
+    function testArgSortDupsInt() public {
```

Comment:
should also test for array with even length

Reply:
ok, will add even length test.